wl/utils/Utils.java

```diff
@@ -83,17 +82,27 @@ public static void log(String msg) {
 		/**
 		 * The read-only website database, which contains details about registered users.
 		 */
-		kWebsite("websitedatabase"),
+		kWebsite("websitedatabase", "websitedbuser", "websitedbpassword"),
 
 		/**
 		 * The add-ons database, which contains moddable metadata about all add-ons.
 		 */
-		kAddOns("addonsdatabase");
+		kAddOns("addonsdatabase", "addonsdbuser", "addonsdbpassword");
 
-		/** The key in the config file that is mapped to this database's name. */
-		public final String configKey;
+		/** The key in the config file that is mapped to this database name. */
+		public final String db_name;
```

Comment:
Stylistic nitpicking: We use CamelCase in this repo, so please rename `dbName`

Reply:
Done